src/hotspot/share/opto/library_call.cpp

```diff
@@ -633,6 +633,8 @@ bool LibraryCallKit::try_to_inline(int predicate) {
     return inline_vector_nary_operation(2);
```

Comment:
Yes, this will be merge with the masked version in future. Since I didn't add the codes for unary/ternary operations, I choose this to be leaved as it is. We can remove it after all opcodes for unary/ternary are supported.